rsocket-examples/src/main/java/io/rsocket/examples/transport/tcp/fnf/TaskProcessingWithServerSideNotificationsExample.java

```diff
@@ -213,7 +213,7 @@ public RSocketTaskHandler(
       @Override
       public Mono<Void> fireAndForget(Payload payload) {
         logger.info("Received a Task[{}] from Client.ID[{}]", payload.getDataUtf8(), id);
-        tasksToProcess.onNext(new Task(id, payload.getDataUtf8()));
+        tasksToProcess.tryEmitNext(new Task(id, payload.getDataUtf8()));
```

Comment:
the `tryEmitNext` method provides immediate feedback and could reject the emission. the return value should be checked, at a minimum, for `isFailed()` status.